### PR TITLE
Feature/support5.2

### PR DESCRIPTION
**Pull request by jimmypuckett**
Updating the package for 5.2

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -55,6 +57,19 @@ Register the HTTP Stack Middleware in file `app/Http/Kernel.php`:
         \Spinen\BrowserFilter\Stack\Filter::class,
 ```
 
+### 5.2
```
Comment (ssfinney) [inline readme.md]: Please be explicit and use: ### Laravel 5.2

Same for 5.1 above.

File: composer.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
         "phpunit/phpunit": "~4.0",
         "psy/psysh": "~0.5.1",
         "satooshi/php-coveralls": "~0.6.1",
-        "symfony/var-dumper": "2.7.*"
+        "symfony/var-dumper": "~2.7|3.0.*"
     },
     "autoload": {
         "files": [
```
Comment (ssfinney) [inline composer.json:35]: Why did you not do a `composer update` to update the composer.lock file?

Comment (jimmypuckett) [inline composer.json:35]: Same as laravel-garbage-man comment.

File: readme.md (hunk 1 above)
Comment (jimmypuckett) [inline readme.md]: Done.

**Pull request merged**